docs/reference/setup/install/deb.asciidoc

```diff
@@ -166,6 +166,8 @@ NOTE: Distributions that use `systemd` require that system resource limits be
 configured via `systemd` rather than via the `/etc/sysconfig/elasticsearch`
 file. See <<systemd>> for more information.
 
+include::get-fingerprint.asciidoc[]
```

Comment:
Maybe call this something different than `get-fingerprint`  ? `connect-client.asciidoc` ?

Reply:
++ I'll rename to `connect-clients.asciidoc`.